tests/git-auto-commit.bats

```diff
@@ -9,10 +9,20 @@ setup() {
     export FAKE_REMOTE="${BATS_TEST_DIRNAME}/tests_remote_repository"
     export FAKE_TEMP_LOCAL_REPOSITORY="${BATS_TEST_DIRNAME}/tests_clone_of_remote_repository"
 
+    # While it is likely the GitHub hosted runners will use master as the default branch,
+    # locally anyone may change that. So for tests lets grab whatever is currently set
+    # globally. This should also ensure that changes to the GitHub hosted runners'
+    # config do not break tests in the future.
+    if [[ -z $(git config init.defaultBranch) ]]; then
+        git config --global init.defaultBranch "main"
+    fi
+
+    export FAKE_DEFAULT_BRANCH=$(git config init.defaultBranch)
```

Comment:
This is great. 👏 
I think on my machine I added a config that repos under the path always use `master` as the default branch. 
As nobody else run these tests before, this didn't come up.$

Thanks for the fix.